spec/lib/jobs_spec.rb

```diff
@@ -88,6 +88,13 @@
       end
     end
 
+    context 'when the search phrase includes "federal" or "gobierno" ' do
+      it 'should not trigger job search' do
```

Comment:
Avoid `should` in spec descriptions:
https://github.com/rubocop-hq/rspec-style-guide#should-in-example-docstrings

`it 'does not trigger a job search' do`